app/src/main/kotlin/com/camerasecuritysystem/client/SettingsActivity.kt

```diff
@@ -125,8 +214,10 @@ class SettingsActivity : AppCompatActivity(),
 
         sharedPreferences.edit().putString(resources.getString(R.string.port), port).apply()
 
-        sharedPreferences.edit().putString(resources.getString(R.string.ip_address), ipAddress).apply()
+        sharedPreferences.edit().putString(resources.getString(R.string.ip_address), ipAddress)
+            .apply()
 
-        sharedPreferences.edit().putString(resources.getString(R.string.camera_id), cameraID).apply()
+        sharedPreferences.edit().putString(resources.getString(R.string.camera_id), cameraID)
+            .apply()
```

Comment:
No functional change...?

Reply:
It was too long for the formatter, it put them on a newline. I will revert this, but the formatter its something we want to use before committing.

Reply:
Done